### ISSUE TITLE
Difficult for new users to find session delete button

### ISSUE THREAD

**Issue by ProCycleDev** [vector-im/element-web#13086]
### Description

When managing a user's session list there is no indication of what the checkbox does or how to remove sessions.

### Steps to reproduce

- Open User Settings
- Go to Security & Privacy section
- Look at Sessions section
- Where is the delete button?

![2020-04-08 14_29_32-Window](https://user-images.githubusercontent.com/18075290/78835873-d34fab80-79a5-11ea-8638-f73469d9f49c.png)

The column containing the checkboxes should have a header to make it clear what the checkbox does and how to delete sessions.

![2020-04-08 14_29_32-Window](https://user-images.githubusercontent.com/18075290/78836612-51608200-79a7-11ea-860e-5425c5e913a9.png)

### Version information

- **Platform**: Desktop
- **OS**: Windows 10
- **Version**: 1.5.15

**Comment by cheryllium** [comment]
I think the reason there isn't a header is because that's where the delete button pops up when you select things: 
![image](https://user-images.githubusercontent.com/5027241/79035051-a14a6f00-7b88-11ea-8754-96bb9e8d4821.png)

Could we move this delete button to a different location, either above or below the list? It's a little clunky in the column and can alter the margins of the page layout when it pops in and out. I think it may look better at the bottom of the list and show a disabled/grayed-out "Delete 0 sessions" when nothing is selected instead of not existing. Then there would be room for the column header (which could also have a counter if wanted)

**Comment by tmysik** [comment]
Sorry for OT but - is it safe to delete sessions? I am asking since I am not sure what happens then with their messages in a room - will they still be there? Will they be still trusted? Does anyone know, please? Cannot find any relevant documentation for it... I want to remove my previous sessions (from my old phone) but not sure whether I can...

Thank you.

**Comment by jryans** [comment]
> is it safe to delete sessions? I am asking since I am not sure what happens then with their messages in a room - will they still be there? Will they be still trusted? Does anyone know, please? Cannot find any relevant documentation for it...

Overall, yes, it's safe and generally good encryption hygiene to delete old sessions. Everyone else in a room has to encrypt their messages for all your sessions, so leaving a bunch of dead ones around causes extra work for others. The new UI coming with cross-signing (will be on release in the next few days) also nudges everyone through toasts to either verify or remove old sessions.

When you remove a session, the messages it sent are still available, but they are marked with a warning shield in the timeline that says "Encrypted by a deleted session".

I'll make a note internally to update our docs with information like this.

**Comment by tmysik** [comment]
@jryans Thank you very much, much appreciated!

**Comment by wget** [comment]
Today, I found this post as my riot client was annoying me on a daily basis asking me to review untrusted sessions. Removing outdated sessions did the trick. I was expecting to have the ability to remove these old sessions from the right sidebar as well and not by having to dig into the advanced settings.

This GitHub issue came in handy and was well indexed on search engines hopefully =) Having a "delete" button appearing when we select a session was hard to guess... indeed.

**Comment by mjg** [comment]
So, what went wrong if there's a list of unverified sessions but no checkboxes at all, so no chance to delete them?

**Comment by t3chguy** [comment]
That you're probably not looking in Settings > Security & Privacy.

**Comment by mjg** [comment]
> That you're probably not looking in Settings > Security & Privacy.

Correctly, I was looking at the profile which has the same list and the verify options but no hint on how to remove the sessions. So, thanks for your hint :)

**Comment by matthijskooijman** [comment]
> I was expecting to have the ability to remove these old sessions from the right sidebar as well and not by having to dig into the advanced settings.

This is the subject of #13187 I think.

**Comment by jryans** [comment]
I expect a lot the feedback here would be addressed by the refreshed devices / sessions settings UI in https://github.com/vector-im/riot-web/issues/11221.

**Comment by erlend-sh** [comment]
> Today, I found this post as my riot client was annoying me on a daily basis asking me to review untrusted sessions. Removing outdated sessions did the trick. I was expecting to have the ability to remove these old sessions from the right sidebar as well and not by having to dig into the advanced settings.

I just ran into exactly this as well. It's a very small fix for what will be a massive time-saving for those who run into this and scratching their head for many bewildered minutes looking for the "Delete session" that should surely be _around here somewhere_ 😄 

Looks like #13187 has correctly diagnosed this issue already and the solution is known. Just a final implementation push still pending.

**Comment by Bugsbane** [comment]
Ok, this is just silly from a UX pov. The first place people become aware of unverified sessions is this alert:
![Screenshot_20210911_162406](https://user-images.githubusercontent.com/401733/132960746-1d6c73e0-ba8c-4e8c-9f81-9ac50cdc79a0.png)
Clicking that takes them to a list where the session can't be deleted. There's no mention they can be managed elsewhere.

The solution seems pretty clear:
![element-verify](https://user-images.githubusercontent.com/401733/132961169-4672511f-ad5a-4859-b54f-cf1fe3b08042.png)

PS. I arrived here after running into the same problem as everyone else, for the same reason, and then doing a search for "Delete unverified matrix sessions".

**Comment by janvlug** [comment]
> is it safe to delete sessions?

I think that this is a very common question that people will have.

> I'll make a note internally to update our docs with information like this.

Please make sure that there is a link to or popup with this information close to the Delete button.

**Comment by aaronraimist** [comment]
Maybe changing the wording from "Delete" to "Sign out" or another word would make it more immediately clear that yes it is safe to do assuming you don't need to use Element from that device anymore.

But yes I agree, having little `i` buttons with a paragraph or two of text explaining features would be nice to have around the UI.

**Comment by rht** [comment]
Bump. I also had to search the internet for how to remove unverified sessions. Reading this thread gave me the knowledge that it is doable from the UI, and I was finally able to find the button to sign out the sessions after scrolling past through >50 old sessions I have accumulated for years.

There should be delete/sign out buttons next to each unverified sessions, in addition to the button that let me do the bulk delete at the end.
